<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D-Matrix Test Point Optimization Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutRoot">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDataset">
       <attribute name="title">
        <string>数据集 (D矩阵)</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutDataset"/>
      </widget>
      <widget class="QWidget" name="tabAlgorithms">
       <attribute name="title">
        <string>算法与单次运行</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutAlgos"/>
      </widget>
      <widget class="QWidget" name="tabBenchmark">
       <attribute name="title">
        <string>Benchmark 对比</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutBench"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
